### PR TITLE
Wrote a release script

### PR DESCRIPTION
**Pull request by melinath**
This is aimed at making semver releases easier.

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+then
+	echo "No version supplied. Run \`make release VERSION=X.X\`"
+	exit 1
+fi
+
+version=$1
+
+echo $version | grep -Eq "^v[0-9]+\.[0-9]+\.[0-9]+$" && echo "match" || echo "no match"
+
+if ! echo $version | grep -Eq "^v[0-9]+\.[0-9]+\.[0-9]+$"
```
Comment (ScottSuarez) [inline release.sh:15]: computed twice. both above and below. Recommend storing in variable

File: release.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+platforms="linux windows darwin"
+build_dir=./bin
+name=terraform-validator
+ldflags="-X github.com/GoogleCloudPlatform/terraform-validator/tfgcv.buildVersion=${version}"
+release_bucket=terraform-validator
+
+# Build release versions
+for platform in ${platforms}; do
+	echo "Building version ${version} for ${platform}..."
+	GO111MODULE=on GOOS=${platform} GOARCH=amd64 CGO_ENABLED=0 go build -ldflags "${ldflags}" -o "${build_dir}/${name}-${platform}-amd64" .
```
Comment (ScottSuarez) [inline release.sh:31]: should we add support for apple silicon or other architectures?

File: release.sh (hunk 1 above)
Comment (melinath) [reply release.sh:15]: Whoops, the upper one was just for testing & I forgot to remove it. Thanks!

File: release.sh (hunk 2 above)
Comment (melinath) [reply release.sh:29]: Possibly? I don't know that we have a clear signal that we're missing particular platforms... But either way, I think that's outside the scope of this PR.

**Review by ScottSuarez — APPROVED** [review]

**Pull request merged**